### ISSUE TITLE
Published files when device reboots

### ISSUE THREAD

**Issue by valllllll2000** [akvo/akvo-flow-mobile#1044]
When the device is restarted while the files were published we need to make sure the files are not left "published".

**Comment by valllllll2000** [comment]
## Test plan
When testing #977 before the 90 minutes are over, restart device. Make sure that the files still disappear once the 90 minutes are over.

**Comment by janagombitova** [comment]
I published, checked the files were shown properly and then restarted my entire device. After it turned back on I checked the app where the publishing count down is still working at should (showing the minutes left - 72 in my case). However when I open the folder the published folder is there with the 2 subfolders but they are empty.

After the time passed, the app updated correctly > Publish data was active again. In the folder structure the folder was there but the zip files got removed correctly as expected.
